libpldmresponder/fru.cpp

```diff
@@ -165,16 +178,19 @@ void FruImpl::buildFRUTable()
     for (const auto& object : objects)
     {
         const auto& interfaces = object.second;
+        bool isPresent = true;
+#ifdef OEM_IBM
+        isPresent =
+            pldm::responder::utils::checkFruPresence(object.first.str.c_str());
```

Comment:
this entire function should ideally sit in the oem code and called from an oem handler, upstream already moved away from using #ifdef OEM in the code base.

Reply:
removed all references to OEM_IBM